packages/gatsby-theme-newrelic/gatsby-node.js

```diff
@@ -278,22 +277,18 @@ exports.onCreatePage = (helpers, themeOptions) => {
 
   if (
     !transformedPage.path.match(/404/) &&
-    transformedPage.context.fileRelativePath.includes('src/pages/')
+    transformedPage.context.fileRelativePath.includes('src/pages/') &&
+    transformedPage.context.locale === 'en'
```

Comment:
this is 🧠  right here. much better than includes 🙈

Reply:
i'm pretty sure you suggested this at some point but i hadn't quite made the connection 😂